src/dbseeder/sql.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
```

Comment:
It's not all about sql. Maybe composition.py or shared.py or common.py. Those are all better than what it is now.